docs/standard/security/index.md

```diff
@@ -1,17 +1,18 @@
 ---
-title: "Security in the .NET Framework"
+title: "Security in .NET"
 ms.date: "03/30/2017"
 ms.technology: dotnet-standard
 helpviewer_keywords: 
-  - ".NET Framework, security"
-  - "security [.NET Framework], about security"
-  - "application development [.NET Framework], security"
+  - ".NET, security"
+  - "security [.NET], about security"
+  - "application development [.NET], security"
   - "security [.NET Framework]"
+  - "security [.NET]"
 ms.assetid: 9a9621d7-8883-4a4f-a874-65e8e09e20a6
 author: "mairaw"
 ms.author: "mairaw"
 ---
-# Security in the .NET Framework
+# Security in .NET
 The common language runtime and the .NET Framework provide many useful classes and services that enable developers to easily write secure code and enable system administrators to customize the permissions granted to code so that it can access protected resources. In addition, the runtime and the .NET Framework provide useful classes and services that facilitate the use of cryptography and role-based security.  
```

Comment:
change description here too and on the link entries below? We can review their content on a later pass.